storage/bootstrap/bootstrapper/fs/source.go

```diff
@@ -38,13 +39,15 @@ type newFileSetReaderFn func(
 	filePathPrefix string,
 	readerBufferSize int,
 	bytesPool pool.CheckedBytesPool,
+	decodingOpts msgpack.DecodingOptions,
 ) fs.FileSetReader
 
 type fileSystemSource struct {
 	opts             Options
 	log              xlog.Logger
 	filePathPrefix   string
 	readerBufferSize int
+	decodingOpts     msgpack.DecodingOptions
```

Comment:
Yeah I already added the decoding options to `FileSystemOptions` but forgot to populate it here, good catch